tests/backup/backup_test.go

```diff
@@ -6488,7 +6488,7 @@ var _ = Describe("{IssueMultipleDeletesForSharedBackup}", func() {
 				backupMap[backupName] = backupUID
 
 				// Start Restore
-				namespaceMapping[bkpNamespaces[0]] = bkpNamespaces[0] + user
+				namespaceMapping[bkpNamespaces[0]] = user
```

Comment:
Can't we just use `bkpNamespaces[0] + "restored"` as the namespace at the destination cluster while restoring?

Reply:
Done